README.md

```diff
@@ -166,10 +191,13 @@ To configure the package behaviour, you can modify your `network_params.json` fi
         //  This mnemonic will a) be used to create keystores for all the types of validators that we have and b) be used to generate a CL genesis.ssz that has the children
         //   validator keys already preregistered as validators
         "preregistered_validator_keys_mnemonic": "giant issue aisle success illegal bike spike question tent bar rely arctic volcano long crawl hungry vocal artwork sniff fantasy very lucky have athlete",
+        // How long you want the network to wait before starting up
+        "genesis_delay": 120,
 
         // The epoch at which the capella and deneb forks are set to occur.
-        "capella_fork_epoch": 2,
-        "deneb_fork_epoch": 4
+        "capella_fork_epoch": 0,
+        "deneb_fork_epoch": 4,
+        "electra_fork_epoch": "None",
```

Comment:
should this be `"None"` or should this be `null` (json)?

Reply:
null json, you are right, fixing it now 

got the None from your mev one `    "mev_type": "None",` Should I use null ?

Reply:
seems like both is fine.

Reply:
The `None` is probably a mistake but if either work I am happy going with `null`